nuxt.config.js

```diff
@@ -57,6 +57,18 @@ export default {
       {
         src: 'https://cloud.ibm.com/analytics/build/bluemix-analytics.min.js',
         async: true
+      },
+      {
+        hid: 'hotjar',
+        innerHTML: `
+(function(h,o,t,j,a,r){
+  h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
+  h._hjSettings={hjid:42920,hjsv:6};
+  a=o.getElementsByTagName('head')[0];
+  r=o.createElement('script');r.async=1;
+  r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
+  a.appendChild(r);
+})(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');`
```

Comment:
What do you think about putting this script in a separate file and add it as plugin in the `nuxt.config.js` as shown [here](https://github.com/nuxt/nuxt.js/issues/2688) based on [this example](https://nuxtjs.org/faq/google-analytics)

Reply:
I think @y4izus 's idea is quite good!